src/util/CommandLine.ts

```diff
@@ -24,4 +24,14 @@ export function commandExecutor(scriptParamName: string) {
     const script = getParameter(scriptParamName);
     if (script === undefined) return undefined;
     return () => console.log(`${scriptParamName}: ${execSync(script).toString().slice(0, -1)}`);
-}
\ No newline at end of file
+}
+
+export function getFloatCommandResult(script: string) {
+    return (parseFloat(`${execSync(script).toString().slice(0, -1)}`) || 0);
```

Comment:
Extract this since this is repeated 3 times in this class

Reply:
ok